scripts/deploy_assisted_service.sh

```diff
@@ -112,13 +115,29 @@ EOF
     echo "Installation of Assisted Install operator passed successfully!"
 else
     print_log "Updating assisted_service params"
+
+    if [ "${DEBUG_SERVICE}" == "true" ]; then
+        print_log "******* Patching assisted service image with a debuggable code ***** "
+        (cd assisted-service/ && skipper --env-file ../skipper.env make update-local-image -e LOCAL_ASSISTED_ORG=$(minikube ip):5000 -e CONTAINER_BUILD_EXTRA_PARAMS="--cgroup-manager=cgroupfs --storage-driver=vfs --events-backend=file")
+    fi
+
     skipper run discovery-infra/update_assisted_service_cm.py
     (cd assisted-service/ && skipper --env-file ../skipper.env run "make deploy-all" ${SKIPPER_PARAMS} $ENABLE_KUBE_API_CMD $OPENSHIFT_VERSIONS_CMD DEPLOY_TAG=${DEPLOY_TAG} DEPLOY_MANIFEST_PATH=${DEPLOY_MANIFEST_PATH} DEPLOY_MANIFEST_TAG=${DEPLOY_MANIFEST_TAG} NAMESPACE=${NAMESPACE} AUTH_TYPE=${AUTH_TYPE})
 
     add_firewalld_port $SERVICE_PORT
 
     print_log "Starting port forwarding for deployment/${SERVICE_NAME} on port $SERVICE_PORT"
-    wait_for_url_and_run ${SERVICE_BASE_URL} "spawn_port_forwarding_command $SERVICE_NAME $SERVICE_PORT $NAMESPACE $NAMESPACE_INDEX $KUBECONFIG minikube"
+    wait_for_url_and_run ${SERVICE_BASE_URL} "spawn_port_forwarding_command $SERVICE_NAME $SERVICE_PORT $NAMESPACE $NAMESPACE_INDEX $KUBECONFIG minikube undeclared $SERVICE_INTERNAL_PORT"
+
+    if [ "${DEBUG_SERVICE}" == "true" ]; then
+        print_log "Removing liveness Probe to prevent rebooting while debugging"
+        kubectl patch deployment assisted-service -n $NAMESPACE --type json   -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/livenessProbe"}]'
+
+        add_firewalld_port $DEBUG_SERVICE_PORT
```

Comment:
Please use parenthesis wherever you can :) 
```suggestion
        add_firewalld_port ${DEBUG_SERVICE_PORT}
```